packages/modal/README.md

```diff
@@ -0,0 +1,104 @@
+# Modal
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/modal.svg)
+
+## Example
+
+```js
+import Modal, { ModalSize, useModalState } from '@leafygreen-ui/Menu';
+
+function ExampleComponent() {
+  const { open, setOpen } = useModalState();
+
+  return (
+    <>
+      <button onClick={() => setOpen(open => !open)}>Open Modal</button>
+      <Modal open={open} setOpen={setOpen}>
+        Modal Content goes here
+      </Modal>
+    </>
+  );
+}
+```
+
+**Output HTML**
```

Comment:
The output here is out-of-date.

In the future, we may wanna hold out on adding this until we finalize the component itself.

Reply:
Yep, got it!